core/server/data/xml/sitemap/base-generator.js

```diff
@@ -82,7 +88,7 @@ class BaseSiteMapGenerator {
         node = {
             url: [
                 {loc: url},
-                {lastmod: moment(this.getLastModifiedForDatum(datum)).toISOString()}
+                {lastmod: moment(this.getLastModifiedForDatum(datum), moment.ISO_8601).toISOString()}
```

Comment:
Don't think `moment.ISO_8601` is necessary here as `toISOString()` [already applies ](https://momentjs.com/docs/#/displaying/as-iso-string/) ISO_8601 formatting automatically

Reply:
Fair enough, seems it can be removed safely. Added it based on its usage in gatsby plugin [here](https://github.com/TryGhost/gatsby-plugin-advanced-sitemap/blob/master/src/BaseSiteMapGenerator.js#L91) 😄